internal/controller/indexer/indexer.go

```diff
@@ -256,6 +267,10 @@ func IngressSecretIndexFunc(rawObj client.Object) []string {
 	return secrets
 }
 
+func IngressHTTPRouteIndexFunc(rawObj client.Object) []string {
```

Comment:
I don't understand the purpose of this index. And it hasn't been used either.

Reply:
Yes, this is useless code, I will delete it.

Reply:
resolved